README.md

```diff
@@ -74,11 +74,29 @@ schedule.
 Meeting notes are available as a public [Google doc][]. If you have trouble
 accessing the doc, please get in touch on [Slack][].
 
-Roles:
+Here is a list of community roles with current and previous members:
 
 - Approvers: [@open-telemetry/docs-approvers][]
+
+  - [Fabrizio Ferri-Benedetti](https://github.com/theletterf), Splunk
+  - [Michael Hausenblas](https://github.com/mhausenblas), Amazon
+
 - Maintainers: [@open-telemetry/docs-maintainers][]
-- Blog approvers: [@open-telemetry/blog-approvers][]
+
+  - [Austin Parker](https://github.com/austinlparker), Honeycomb
+  - [Patrice Chalin](https://github.com/chalin), CNCF
+  - [Phillip Carter](https://github.com/cartermp), Honeycomb
+  - [Severin Neumann](https://github.com/svrnm), Cisco
+
+- Emeritus approvers:
+
+  - [Paul Bruce](https://github.com/paulsbruce), Tricentis
+
+- Emeritus maintainers:
+
+  - [Steve Flanders](https://github.com/flands), Splunk
+  - [Morgan McLean](https://github.com/mtwo), Splunk
```

Comment:
Typically, for Emeritus there is no need to keep company. It can be easily outdated.

```suggestion
  - [Paul Bruce](https://github.com/paulsbruce)

- Emeritus maintainers:

  - [Steve Flanders](https://github.com/flands)
  - [Morgan McLean](https://github.com/mtwo)
```

Reply:
good call out, thanks!